src/error/views/ErrorView.tsx

```diff
@@ -71,9 +71,12 @@ const ErrorView: FunctionComponent<ErrorViewProps & RouteComponentProps & UserPr
 	}
 
 	const messageText = (queryParams.message as string) || message || '';
-	const errorMessage: string = isNil(messageText)
-		? i18n.t('error/views/error-view___de-pagina-werd-niet-gevonden')
-		: messageText;
+	let errorMessage: string;
```

Comment:
Is dit niet gewoon

`const errorMessage = isNil(messageText) ? getPageNotFoundError(!!user) : messageText;`

Reply:
yep